bin/deploy-vhost

```diff
@@ -135,16 +135,30 @@ sub get_pgpw_string {
 
 sub one_db_conf_php {
     my ($params, $database, $internal) = @_;
-    my $port_line = '';
+    my $port_line       = '';
+    my $db_ro_host_line = '';
+    my $port_ro_line    = '';
     if ($params->{type} eq 'psql') {
         $port_line = "define('OPTION_$params->{prefix}_DB_PORT', $params->{port});";
+        if ($params->{replica}) {
+            $db_ro_host_line = "define('OPTION_$params->{prefix}_DB_RO_HOST', '$params->{replica}.ukcod.org.uk');";
```

Comment:
For consistency (though I realise it's the same output), should this be `get_db_host($params->{replica})`?

Reply:
Is then removed by 2nd commit.